packages/composite-checkout/src/custom.d.ts

```diff
@@ -0,0 +1,7 @@
+import '@emotion/react';
+import type { Theme as CheckoutTheme } from './lib/theme';
+
+declare module '@emotion/react' {
+	// eslint-disable-next-line @typescript-eslint/no-empty-interface
+	export interface Theme extends CheckoutTheme {}
+}
```

Comment:
What is the reason for adding the `Theme` interface?

Also, isn't the `@emotion/react` module already been "augmented" in `packages/composite-checkout/src/types.ts` ?

Reply:
> Also, isn't the @emotion/react module already been "augmented" in packages/composite-checkout/src/types.ts ?

Great question! That augmentation is actually the old way of defining themes. The new recommended way is to augment the theme type like here, and then we can import directly from `@emotion/styled` instead of importing from a custom styled file. More here: https://emotion.sh/docs/emotion-11#theme-type

The `styled` files should be deleted in this PR as well

Reply:
I think the concept is that `@emotion/` infers the correct `Theme` type internally instead of using a custom type everywhere we use `styled`